modules/persistent-storage-csi-olm-driver-install.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc
+
+[id="persistent-storage-csi-olm-driver-install_{context}"]
+= Installing the {FeatureName} CSI driver
+
+The {FeatureName} CSI driver is not installed in {product-title} by default. Use the following procedure to install and configure this driver to enable the {FeatureName} CSI Driver Operator in your cluster.
```

Comment:
enable the {FeatureName} CSI Driver Operator -> enable the {FeatureName} CSI Driver
(the operator is already installed at this point)